### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+# TPM2 module schematics
 
-TPM module PCB project for Libretrend LT1000
\ No newline at end of file
+TPM2 module PCB project for platforms with 2x5 pin, 2mm pitch LPC header, e.g.
+Libretrend LT1000.
+
+## Description
+
+Trusted Platform Module2 compatible with 2x5p LPC header (populated on Librebox
```
Comment (miczyg1) [inline README.md]: Just Trusted Platform Module, without '2'.

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+* Meeting Intel TXT, Microsoft Windows and Google Chromebook certification
+  criteria for successful platform qualification
+* True Random Number Generator (TRNG)
+* Full personalization with Endorsement Key (EK) and EK certificate
+* Supports the LPC interface
+
+## Where I can buy this product?
+
+If you are not the enthusiast of DYI, then the plug and play product can be
+bought in the [3mdeb shop](https://shop.3mdeb.com/product/tpm2-module-for-librebox/).
```
Comment (pietrushnic) [inline README.md]: @artur-rs question if Libretrend selling this module - we should add link.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
-# tpm_module
+# TPM2 module schematics
 
-TPM module PCB project for Libretrend LT1000
\ No newline at end of file
+TPM2 module PCB project for platforms with 2x5 pin, 2mm pitch LPC header, e.g.
+Libretrend LT1000.
```
Comment (pietrushnic) [inline README.md]: @artur-rs I would definitely write that this TPM was created for Librebox and under contract work from Libretrend. Let's avoid any confusion that we did it on our own without being compensated.

Also, the question is if pin layout is important in this case? Or it is always the same? If it is uniq we should probably elaborate that and point to graphics. BTW I think image of pin layout should be here and attached to social media announcement.

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (artur-rs) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (artur-rs) [reply README.md]: @pietrushnic please check if now is ok

**Review by pietrushnic — APPROVED** [review]

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**